main.cpp

```diff
@@ -41,22 +41,458 @@ If you need inspiration for what to write, take a look at previously approved st
 /*
  UDT 1:
  */
+#include <iostream>
+#include <string>
+#include <cmath>
+
+using namespace std;
+
+struct Synth
+{
+    struct UIControls
+    {
+        UIControls(int Knob_, int Button_) : Knob(Knob_), Button(Button_)
```

Comment:
no underscores in your variable names.
Variable names should always begin with a lowercase letter.  Capital Letters are reserved for Class names and Static class members
- [ ] revise